app/models/solr_document.rb

```diff
@@ -24,6 +24,20 @@ class SolrDocument
   # Recommendation: Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
 
+  module TeiDocument
+    def self.extended(document)
+      document.will_export_as(:tei, 'application/tei+xml')
+    end
+
+    def export_as_tei
+      self[:tei_ss]
+    end
+  end
+
+  use_extension(TeiDocument) do |document|
+    document.has? :tei_ss
```

Comment:
![oh you fancy huh](https://media1.tenor.com/images/ae10a72f76b3243e76e481405c4b163c/tenor.gif)

Reply:
![](https://media.giphy.com/media/RddAJiGxTPQFa/giphy.gif)

Reply:
![awww yisssss](https://media.giphy.com/media/pa37AAGzKXoek/giphy.gif)